### ISSUE TITLE
Recommendations don't work in Firefox

### ISSUE THREAD

**Issue by Lyckra** [syndiate/Reconstruct-Classic-Watch8-Layout#2]
Recommendations don't work at all in Firefox and because of that the script fails to load half of the time. The other half it loads without recommendations.

**Comment by syndiate** [comment]
Yes. I'm aware of its loading issues. It's a bug testing build for a reason
